src/components/Navigation/HamburgerMenu.jsx

```diff
@@ -0,0 +1,9 @@
+import {Sling as Hamburger} from 'hamburger-react';
+
+export default function HamburgerMenu() {
+  return (
+    <>
+      <Hamburger />
+    </>
```

Comment:
You don't need the fractions here as `Hamburger` is your only return value
```suggestion
      <Hamburger />
```